exist-core/src/main/java/org/exist/storage/BrokerPool.java

```diff
@@ -1095,11 +1095,25 @@ public PerformanceStats getPerformanceStats() {
      * Returns a pool in which the database instance's readers are stored.
      *
      * @return The pool
+     *
+     * @deprecated Use {@link #getXmlReaderPool()} instead
      */
+    @Deprecated
     public XMLReaderPool getParserPool() {
         return xmlReaderPool;
     }
 
+    /**
+     * Returns a pool in which the database instance's readers are stored.
+     *
+     * @return The pool
+     *
+     * @deprecated Use {@link #getXmlReaderPool()} instead
```

Comment:
I wonder if this should not have the deprecated field in the docblock referencing itself. At least it is confusing me.